File: app/src/main/java/io/github/vvb2060/keyattestation/home/HomeActivity.java
package io.github.vvb2060.keyattestation.home;

import android.os.Bundle;
import android.view.Menu;

import io.github.vvb2060.keyattestation.R;
import io.github.vvb2060.keyattestation.app.AppBarFragmentActivity;

public class HomeActivity extends AppBarFragmentActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.fragment_container, new HomeFragment())
                    .commit();
        }
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.home, menu);
        return true;
    }
}

File: app/src/main/java/io/github/vvb2060/keyattestation/home/HomeActivity.kt
package io.github.vvb2060.keyattestation.home

import android.os.Bundle
import androidx.fragment.app.commit
import io.github.vvb2060.keyattestation.BuildConfig
import io.github.vvb2060.keyattestation.R
import io.github.vvb2060.keyattestation.app.AppBarFragmentActivity

class HomeActivity : AppBarFragmentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportActionBar?.subtitle = BuildConfig.VERSION_NAME

        if (savedInstanceState == null) {
            supportFragmentManager.commit {
                setReorderingAllowed(true)
                add(R.id.fragment_container, HomeFragment())
            }
        }
    }
}
